### PR TITLE
Javascript typing tutor - Scarlet Kim

### PR DESCRIPTION
**Pull request by Scarlet-K**
typing

### CODE REVIEW

File: javascript-typing-tutor/main.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+function handleKeydown(event) {
+  if (($characters[counter].textContent === event.key) && (counter !== $characters.length - 1)) {
+    $characters[counter].className = 'correct';
+    $characters[counter + 1].className = 'border-bottom';
+    counter++;
+  } else if (($characters[counter].textContent !== event.key) && (counter !== $characters.length - 1)) {
+    $characters[counter].className = 'incorrect';
+  } else if (counter === $characters.length - 1) {
+    event.preventDefault();
+  }
```
Comment (codingcodymiller) [inline javascript-typing-tutor/main.js:20]: What happens if you delete this block?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-typing-tutor/main.js (hunk 1 above)
Comment (Scarlet-K) [reply javascript-typing-tutor/main.js:20]: I get an error message if the user continues to type after the last space, so I thought I should "end" the event. And this was the only method I could think of!

Comment (Scarlet-K) [reply javascript-typing-tutor/main.js:20]: last space = last character*

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**